admins/pageflow/entry.rb

```diff
@@ -130,6 +130,18 @@ module Pageflow
                   input_html: {style: 'width: 200px'})
         end
 
+        if f.object.new_record?
+          col = entry_type_collection
+          if col.length > 1
+            f.input :type_name,
+                    as: :select,
+                    include_blank: false,
+                    collection: col
+          else
+            f.hidden_field :type_name, value: col.first.first
+          end
+        end
+
```

Comment:
This causes the select to be shown soon as a second entry type is registered in the system. We could consider, hiding the select via JS when it only contains a single item. This would let us install the scrolled entry type behind a disabled feature flag without letting our customers know that entry types are about to be a thing. Also we could simplify the branching logic here. Toggling between accounts with different available entry types would then make the select show up and disappear as well, which might be a bit more confusing than showing a disabled select.

Reply:
The 'nicest' solution that I'm currently thinking of from a UX perspective would be to only hide the select for users who never have more than one option for the entry type at a time. However, I'm assuming that in terms of complexity that would be excessive. In case you'd think it's okay, I'd go with that, though.

Without further input, I'll just go with hide instead of disable

Reply:
Agreed. The simple solution of hide instead disable sounds fine for now.